mofacts/server/methods.js

```diff
@@ -713,19 +713,15 @@ async function setExperimentState(UserId, TDFId, newExperimentState) { // by cur
   return TDFId;
 }
 
-async function insertHiddenItem(userId, clusterKC) {
-  let query = 'UPDATE componentstate SET showitem = FALSE WHERE userid = $1 AND kcid = $2';
-  await db.manyOrNone(query, [userId, clusterKC]);
+async function insertHiddenItem(userId, stimulusKC, tdfId) {
+  let query = 'UPDATE componentstate SET showitem = FALSE WHERE userid = $1  AND tdfid = $2 AND kcid = $3 AND componenttype = \'stimulus\'';
```

Comment:
Prefer to do double quotes when inside single quotes instead of escaping

Reply:
Turns out to do this you have to put the whole query in double quotes and then the string parameter in single quotes.